tests/figures/chart/waterfall/waterfall_panel_component.test.ts

```diff
@@ -91,7 +99,7 @@ describe("Waterfall chart side panel", () => {
       await openChartConfigSidePanel(model, env, chartId);
       await click(fixture, ".o-panel-design");
 
-      expect(getHTMLInputValue(".o-chart-title input")).toEqual("My Waterfall chart");
+      expect(textContent(".o-chart-title .o-composer")).toEqual("My Waterfall chart");
```

Comment:
ADRM introduced `toHaveText` which what you cant here ;)
```suggestion
      expect(".o-chart-title .o-composer").toHaveText("My Waterfall chart");
```